druid/src/core.rs

```diff
@@ -975,7 +975,6 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                     self.state.needs_window_origin = false;
                     true
                 }
-                #[cfg(test)]
```

Comment:
did you consider putting this behind a feature? e.g. this would be `#[cfg(any(test, feature="test-harness"))]`? This might cause minor problems downstream if anyone is trying to exhaustively handle events anywhere.

Reply:
Yeah, I did consider it, and yes, the added complexity and risk for breakage is a concern. I've ranted about cfg not being great before.

I guess we could simply mark the `InternalLifeCycle` enum as `#[non_exhaustive]`.

I'm very on the fence on whether adding a feature gate is worth the hassle (linkers will remove most unused code anyway), but I'm willing to add it as long as I don't have to update the CI and stuff myself.